### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by Lawrencerichards**
Made a gov uk coded version of the homepage and made the changes mentioned in user testing sessions for example:

- Rephrased the top links to be more user friendly
- added who the NSC is on top links
- changed the logo to a crest style (screen readable)
- moved a lot of the content into seperate pages (under organisational information)
- made the 'what the NSC does' section more layman friendly, and made links sentences rather than just words

Screen Shot 2019-05-17 at 09 04 58

Added link on carousel page to new homepage

Added images to assets

### CODE REVIEW

File: app/views/topup/homepage.html [hunk 1]
```diff
@@ -0,0 +1,656 @@
...
+    </div>
+
+      <div class="column-one-third organisation__margin-bottom">
+
+          <ol class="app-c-topic-list app-c-topic-list--small brand--department-for-business-innovation-skills ">
+            <li class="app-c-topic-list__item">
+              <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="#role">What the UK National Screening Committee does</a>
+            </li>
+      <li class="app-c-topic-list__item">
+        <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="https://www.gov.uk/guidance/nhs-population-screening-explained">Screening explained</a>
```
Comment (NeilvB) [inline app/views/topup/homepage.html]: We had assessment feedback on the homepage which said 'Link headings do not match page headings' - should this be 'NHS Population Screening Explained' ?

File: app/views/topup/homepage.html [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+      </li>
+      <li class="app-c-topic-list__item">
+        <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="/search">Current screening guidance pages</a>
```
Comment (NeilvB) [inline app/views/topup/homepage.html]: Thinking in a similar way to the above - should this be 'Complete list of UK NSC recommendations' ?

File: app/views/topup/homepage.html [hunk 3]
```diff
@@ -0,0 +1,656 @@
...
+      </li>
+      <li class="app-c-topic-list__item">
+        <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="https://phescreening.blog.gov.uk/">Screening blog</a>
+      </li>
+      <li class="app-c-topic-list__item">
+        <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="https://www.gov.uk/topic/population-screening-programmes">Screening programmes across the UK</a>
```
Comment (NeilvB) [inline app/views/topup/homepage.html]: Also thinking along simiilar lines - should this be 'Population screening programmes' ?

File: app/views/topup/homepage.html (hunk 1 above)
Comment (Lawrencerichards) [reply app/views/topup/homepage.html]: Aw but I really want to change them! It feels like from the perspective of an 'infrequent visitor' to the site, the links don't make much sense as they currently are. Anyone who is used to the site 'admin' or 'policy stakeholder' doesn't have a problem. I guess we were much more exposed to infrequent visitors this time round and it was really obvious that the links didn't mean much without context. I also get the importance of the assessors comments but also, that they don't have the project context. I wonder if it's gds content design policy to have same link headings or if there is some leeway? I think i will ask Iain first and see what he says (pretty sure he advocated making them less jargon-y!)

File: app/views/topup/homepage.html (hunk 2 above)
Comment (Lawrencerichards) [reply app/views/topup/homepage.html]: Also just thinking that maybe there is scope at least on the portion we are building to actually change the titles of the pages? i.e not sure if 'complete list of UK NSC recommendations' is set in stone

**Pull request merged**